LongWarOfTheChosen/Localization/LW_PerkPack_Integrated/XComGame.rus

```diff
@@ -1484,5 +1484,4 @@ RandomNicknames_Female[4]="Воительница"
 RandomNickNames_Female[5]="Валькирия"
 ;ALPHABETICAL ORDER
 
-[X2Ability_PerkPackAbilitySet]
-LocCoveringFireMalus="По этому юниту был произведен выстрел из режима наблюдения со способностью \"Огневая поддержка\", дающий штраф -10 к меткости до конца хода."
```

Comment:
It looks like this should be moved to *LW_PerkPack_Integrated.rus* rather than just deleted. See *LW_PerkPack_Integrated.int*.

Reply:
done